### PR TITLE
HS-859: operator will ping the UI as a health check on the instance

### PR DESCRIPTION
**Pull request by jkapponms**
## Description
Operator will send a request to the UI service, and will mark the instance as ready and healthy if the UI responds with a 200 OK. 

## Jira link(s)
- https://issues.opennms.org/browse/HS-859

### CODE REVIEW

File: operator/internal/reconciler/health_check_test.go [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+			},
+			Release: values.HelmRelease{
+				Namespace: namespace,
+			},
+		},
+	}
+	return instance
+}
+
+func Test_InstanceReady_200OK(t *testing.T) {
```
Comment (kbakanov) [inline operator/internal/reconciler/health_check_test.go:69]: Looks good, but please add precondition and expected result, e.g. Test_InstanceReady_LoginAvailable_ReturnsTrue

Comment (kbakanov) [reply operator/internal/reconciler/health_check_test.go:69]: The same goes for other ones too

Comment (kbakanov) [reply operator/internal/reconciler/health_check_test.go:69]: Btw, probably best to use lower case "i", i.e. instanceReady instead of InstanceReady.

**Comment by jkapponms** [comment]
I'll work on splitting the test methods up in other parts of the code base as I return to them.

**Review by kbakanov — APPROVED** [review]

**Pull request merged**